migrations/groups/add-unlimited-subscription.js

```diff
@@ -0,0 +1,25 @@
+import { model as Group } from '../../website/server/models/group';
```

Comment:
Could we throw in a comment explaining the purpose of each migration? Like, I have a guess that this migration is for giving groups free indefinite group plans (for an admin perk, or testing, etc.), but others visiting the repo might not immediately grasp that.